extensions/amp-bind/0.1/bind-evaluator.js

```diff
@@ -99,17 +99,18 @@ export class BindEvaluator {
    */
   evaluateBindings(scope) {
     /** @type {!Object<string, ./bind-expression.BindExpressionResultDef>} */
-    const cache = {};
+    const cache = Object.create(null);
     /** @type {!Object<string, !EvaluatorErrorDef>} */
-    const errors = {};
+    const errors = Object.create(null);
 
+    // First, evaluate all of the expression strings in the bindings.
     this.bindings_.forEach(binding => {
-      const {tagName, property, expressionString} = binding;
+      const expressionString = binding.expressionString;
       // Skip if we've already evaluated this expression string.
       if (cache[expressionString] !== undefined || errors[expressionString]) {
         return;
       }
-      const expression = this.expressionCache_[expressionString];
+      const expression = this.expressions_[expressionString];
       if (!expression) {
```

Comment:
Just curious, under what circumstances will the cache not have an expression at this point?